PresContest/src/org/icpc/tools/presentation/contest/internal/tile/TeamTileHelper.java

```diff
@@ -210,7 +210,7 @@ private void paintName(Graphics2D g, ITeam team, int ww, int maxwid) {
 
 			nameImages.put(hash, img);
 		}
-		g.drawImage(img, ww + tileDim.height + IN_TILE_GAP - 1, tileDim.height * 1 / 10 - 2, maxwid + 2, img.getHeight(),
+		g.drawImage(img, ww + tileDim.height + IN_TILE_GAP - 1, tileDim.height * 1 / 10 - 1, maxwid + 2, img.getHeight(),
 				null);
```

Comment:
Matching
```java
g.drawImage(img, ww + tileDim.height + IN_TILE_GAP - 1, tileDim.height * 1 / 10 - 1, null);
```
in [TeamTileHelper.java#L173](https://github.com/icpctools/icpctools/blob/main/PresContest/src/org/icpc/tools/presentation/contest/internal/tile/TeamTileHelper.java#L173)